dbt_server/views.py

```diff
@@ -195,7 +196,7 @@ def parse_project(args: ParseArgs):
 
 @app.post("/async/dbt", response_model=schemas.Task)
```

Comment:
```suggestion
@app.post("/async/dbt")
```
Removing the `response_model` as this PR changes the response structure